starter_code/css/style.css

```diff
@@ -64,59 +62,51 @@ a {
 
 /* Center the container */
 .container {
-  width: 960px;
+  max-width: 960px;
   margin: 0 auto;
 }
 .logo {
-  font-size: 48px;
   font-weight: 800;
 }
 
 header {
   background: #033048;
-  padding: 25px 0;
   overflow: hidden;
 }
+
 header .logo {
   color: #fff;
   float: left;
 }
 
 header nav {
   float: right;
```

Comment:
Are you referring to the floats? Wont they automatically be applied by the base styles.